### ISSUE TITLE
playground freezes browser

### ISSUE THREAD

**Issue by DetachHead** [ota-meshi/eslint-plugin-svelte#317]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

n/a

### What version of `eslint-plugin-svelte` are you using?

n/a

### What did you do?

navigate to https://ota-meshi.github.io/eslint-plugin-svelte/playground/#eJyrrgUAAXUA+Q==

### What did you expect to happen?

it works

### What actually happened?

![image](https://user-images.githubusercontent.com/57028336/206986083-d462ff8a-2375-47fa-bca1-bf70449a318a.png)
![image](https://user-images.githubusercontent.com/57028336/206986102-fc83e03e-5288-4084-99e4-62e9968593a3.png)

### Link to **GitHub Repo** with Minimal Reproducible Example

n/a

### Additional comments

tested on chrome and firefox on 3 different devices

**Comment by ota-meshi** [comment]
Hmm. It looks like the postcss tokenizer source code is broken by minify. Broken postcss tokenizer loops infinitely.

**Comment by ota-meshi** [comment]
Hmm. Even with minify turned off, the necessary source code is missing in the build. I still don't know the cause of it.

Original:

```js
          RE_WORD_END.lastIndex = pos + 1
          RE_WORD_END.test(css)
          if (RE_WORD_END.lastIndex === 0) {
            next = css.length - 1
          } else {
            next = RE_WORD_END.lastIndex - 2
          }
```

Output:

```js
          RE_WORD_END.lastIndex = pos + 1;
          if (RE_WORD_END.lastIndex === 0) {
            next = css.length - 1;
          } else {
            next = RE_WORD_END.lastIndex - 2;
          }
```

**Comment by ota-meshi** [comment]
I still don't know the cause, so I opened an issue on vite's repository.
https://github.com/vitejs/vite/issues/11330

**Comment by DetachHead** [comment]
if it helps narrow it down, i'm pretty sure it was working properly until a few days ago

**Comment by ota-meshi** [comment]
I suspect that a major version upgrade of vite v4 or rollup v3 is the cause of the problem, but I'm not sure how to confirm that at the moment :sweat_smile:
